src/kivy_state.py

```diff
@@ -0,0 +1,72 @@
+"""
+Kivy_state.py
+=================================
+"""
+import collections
+
+
+# Network protocols availability, initialised below
+networkProtocolAvailability = None
+
+sqlReady = False  # set to true by sqlTread when ready for processing
+
+trustedPeer = None
+Peer = collections.namedtuple('Peer', ['host', 'port'])
+
+
+def resetNetworkProtocolAvailability():
+    """This method helps to reset the availability of network protocol"""
+    # pylint: disable=global-statement
+    global networkProtocolAvailability
```

Comment:
if we need to modify this, maybe make `networkProtocolAvailability` into an object? And make `reset` a method of this object.